### PR TITLE
feat(ghosting): sync interactions across browsers

### PR DESCRIPTION
**Pull request by vobu**

### CODE REVIEW

File: packages/ui5-app/package.json [hunk 1]
```diff
@@ -32,9 +32,11 @@
     "@openui5/sap.m": "^1.71.0",
     "@openui5/sap.ui.core": "^1.71.0",
     "@openui5/sap.ui.layout": "^1.71.0",
+    "@openui5/sap.ui.unified": "^1.72.1",
```
[the remark this answers is not in the document]
Comment (petermuessig) [reply packages/ui5-app/package.json]: BTW: Unified isn't used anymore as no YearPicker is used - right?

File: packages/ui5-middleware-iasync/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+npm install ui5-middleware-iasync --save-dev
+```
+
+## Configuration options (in `$yourapp/ui5.yaml`)
+
+- `https: <boolean>` default: false  
+whether to use the middleware via SSL/wss
+- `httpModule: <string>`, default: undefined  
+capability to e.g. use `http2`
+- `port: <integer>`, default: 3000
```
Comment (petermuessig) [inline packages/ui5-middleware-iasync/README.md]: shouldn't we change the default? Does this collide with any other port?

File: packages/ui5-app/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (vobu) [reply packages/ui5-app/package.json]: aligned versions + kicked sap.ui.unified with last commit

File: packages/ui5-app/package.json [hunk 3]
```diff
@@ -32,9 +32,11 @@
...
     "@openui5/themelib_sap_fiori_3": "^1.71.0"
   },
   "devDependencies": {
+    "@openui5/ts-types": "^1.60.5",
```
[the remark this answers is not in the document]
Comment (vobu) [reply packages/ui5-app/package.json]: now added tsconfig.json in latest commit - forgot that one previously

File: packages/ui5-app/package.json [hunk 4]
```diff
@@ -45,6 +47,7 @@
     "npm-watch": "^0.6.0",
     "rimraf": "~3.0.0",
     "ui5-middleware-cfdestination": "^0.1.3",
+    "ui5-middleware-iasync": "0.0.1-alpha",
```
[the remark this answers is not in the document]
Comment (vobu) [reply packages/ui5-app/package.json]: done

File: packages/ui5-middleware-iasync/README.md (hunk 2 above)
Comment (vobu) [reply packages/ui5-middleware-iasync/README.md]: browsersync default is 3000 - currently no conflict

Comment (petermuessig) [reply packages/ui5-middleware-iasync/README.md]: Ok

File: packages/ui5-app/package.json [hunk 5]
```diff
@@ -56,9 +58,11 @@
       "@openui5/sap.ui.core",
       "@openui5/sap.m",
       "@openui5/sap.ui.layout",
+      "@openui5/sap.ui.unified",
```
[the remark this answers is not in the document]
Comment (vobu) [reply packages/ui5-app/package.json]: removed

File: packages/ui5-app/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
```
[the remark this answers is not in the document]
Comment (vobu) [reply packages/ui5-app/tsconfig.json]: ts-types still used in ui5-app!

**Pull request merged**